.github/workflows/docker-build-static.yml

```diff
@@ -453,14 +463,18 @@ jobs:
           # list of Docker images to use as base name for tags
           images: |
             nitnelave/lldap
+            ghcr.io/lldap/lldap
```

Comment:
```suggestion
            lldap/lldap
            ghcr.io/lldap/lldap
```

Reply:
do we already had `lldap` on docker? as `lldap/lldap` will lead to `index.docker.io/lldap/lldap`

Reply:
docker pull works for me with it

Reply:
I think we still want nitnelave/lldap since that's waht a LOT of people target still